site_preference_builder/builder.js

```diff
@@ -286,3 +299,89 @@ exports.addCustomGroupFromPlan = (file, plans) => {
 
   return file;
 };
+
+exports.addRefundCustomAttributes = (file) => {
```

Comment:
The builder was intended for the site preference attributes. Now it's getting bigger and the aim of this file is getting lost. Let's refactor/split it a bit.
The part you want to add only concern the refund feature.
We also have the `onShipment` file about the jobs.
I think the sensible thing to do would be to have a `/jobs` folder, that will include a `onShipment` and a `refund` file.
what do you think about it ?